lib/peluquera.ex

```diff
@@ -41,6 +46,11 @@ defmodule Peluquera do
 
   ##############################################################################
 
+  defp add_rabbit_conf_to_peluqueria([], {exchg, conf}), do: {exchg, conf}
+
+  defp add_rabbit_conf_to_peluqueria(rabbit, {exchg, conf}),
+    do: {exchg, Keyword.merge(conf, rabbit: rabbit)}
+
```

Comment:
In `peluqueria.ex:111` we have `opts[:rabbit] || @rabbit` so with default `nil` it’s safe to add `rabbit:` keyword no matter what value we have here.

```suggestion
  defp add_rabbit_conf_to_peluqueria(rabbit, {exchg, conf}),
    do: {exchg, Keyword.merge(conf, rabbit: rabbit)}
```

Reply:
If `nil == Application.get_env(:peluquero, :rabbit, nil)` then it overrides rabbits nested configuration. Thus, eliminating the possibility of having independent rabbits (if that ever happens, but we don't know)